ui/lib/apps/Statement/pages/List/StatementSettingForm.tsx

```diff
@@ -186,9 +185,9 @@ function StatementSettingForm({ onClose, onConfigUpdated }: Props) {
                       </Input.Group>
                     </Form.Item>
                     <Form.Item label={t('statement.settings.keep_duration')}>
-                      <Typography.Text disabled>
+                      <span style={{ color: '#555' }}>
```

Comment:
I think it would be better to avoid this hardcoding.. Why not just using the default color?

Reply:
yep, it seems not good, just want to differ the item label and content a bit.